sim/common/sod/item_effects/phase_5.go

```diff
@@ -161,25 +185,122 @@ func init() {
 		})
 	})
 
-	core.AddEffectsToTest = true
-}
+	// https://www.wowhead.com/classic/item=231784/lightnings-cell
```

Comment:
Looks like this was copy-pasted and not updated?

Reply:
updated